src/components/PackTile.js

```diff
@@ -87,12 +91,13 @@ const PackTile = ({
           object-fit: scale-down;
           width: ${view === VIEWS.GRID ? 100 : 25}%;
           padding: 0 ${view === VIEWS.GRID ? 5 : 1}%;
-          margin: ${view === VIEWS.GRID ? 'auto' : 0};
+          margin: 10px auto 10px auto;
```

Comment:
This could be simplified to:
```suggestion
          margin: 10px auto;
```